ubl/Handlers/Invoice/InvoiceLineHandler.php

```diff
@@ -152,4 +164,20 @@ public function handle( $data, $options = array() ) {
 		return $data;
 	}
 
+	/**
+	 * Get the unit price of an item
+	 *
+	 * @param WC_Order_Item $item
+	 * @return int|float
+	 */
+	private function get_item_unit_price( $item ) {
+		if ( is_a( $item, 'WC_Order_Item_Product' ) ) {
+			return $item->get_subtotal() / $item->get_quantity();
```

Comment:
Maybe you can use `get_item_subtotal()` method?
`$this->document->order->get_item_subtotal( $item, false, true );`

Reply:
Good catch!

Reply:
Yes, it's the same. The only advantage might be using the provided source function or the applied filter. If those don't matter, the current code is fine, as it doesn't change over time.

Reply:
This seems good for now, and we can always tweak it later!